src/main/java/teammates/common/datatransfer/FeedbackResponseCommentAttributes.java

```diff
@@ -89,6 +95,8 @@ public FeedbackResponseCommentAttributes(FeedbackResponseComment comment) {
         this.commentText = comment.getCommentText();
         this.giverSection = comment.getGiverSection() != null ? comment.getGiverSection() : "None";
         this.receiverSection = comment.getReceiverSection() != null ? comment.getReceiverSection() : "None";
+        this.lastEditorEmail = comment.getLastEditorEmail();
+        this.lastEditedAt = comment.getLastEditedAt();
```

Comment:
this 2 attributes will be null for legacy comments and 
will be dangerous for other developers. it needs a default value instead